pkgs/love2d/example/assets/relic_breach/art/README.md

```
@@ -0,0 +1,9 @@
+Asset packs will be unpacked here.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)